src/org/zaproxy/zap/extension/scripts/ExtensionScriptsUI.java

```diff
@@ -721,6 +838,35 @@ public void run() {
 			}
 		}
 	}
+	
+	private void installExtenderScript(ScriptWrapper script) {
+		ExtenderScript ec;
+		try {
+			ec = extScript.getInterface(script, ExtenderScript.class);
+			ec.install(getExtensionScriptHelper());
+			this.installedExtenderScripts.put(script.getName(), ec);
+			script.setError(false);
+		} catch (Exception e) {
+			LOGGER.warn("Failed to install extender script " + script.getName(), e);
+			script.setError(true);

```

Comment:
Could call `extScript.setError(script, e)` instead? (or that causes problems?) (same when uninstalling)